components/ui/themes/DefaultTheme.ts

```diff
@@ -529,14 +529,28 @@ export const DefaultTheme = {
       width: 70,
       borderRadius: 200,
       padding: 10,
-      backgroundColor: Colors.Orange,
       shadowColor: '#000',
       shadowOpacity: 0.4,
       elevation: 5,
       position: 'absolute',
       bottom: Dimensions.get('window').width * 0.1,
       right: Dimensions.get('window').width * 0.1,
     },
+    fabIconNormal: {
```

Comment:
Can we download  / plus word also here to be more explicit about this icon use-case (download card button)? like downloadFabIconNormal

Reply:
Updated it.